pyproject.toml

```diff
@@ -114,6 +114,9 @@ test = [
 ]
 dev = ["build", "pre-commit", "pytest-cov", "coverage", "hatch", "bump2version"]
 
+[tool.check-wheel-contents]
+ignore = ["W002", "W004"]
```

Comment:
Maybe we could add those to the cookiecutter by default now that it has been removed from the default releaser check.

Reply:
Yes - actually only W002 is needed.

Reply:
Opened https://github.com/jupyterlab/extension-cookiecutter-ts/pull/275